target/linux/ipq806x/patches-4.19/0063-3-tsens-fix-kernel-4_19.patch

```diff
@@ -0,0 +1,20 @@
+--- a/drivers/thermal/qcom/tsens-common.c
++++ b/drivers/thermal/qcom/tsens-common.c
+@@ -126,6 +126,7 @@ static const struct regmap_config tsens_
+ 
+ int __init init_common(struct tsens_device *tmdev)
+ {
++	resource_size_t size;
```

Comment:
I just noticed this, sorry for this nitpick. I would move the definition under `struct resource *res;`, to keep christmast tree ordering.